web/html/src/manager/systems/appstreams/ssm-appstreams-channel-selection.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+
+import { Column } from "components/table/Column";
+import { HierarchicalRow, HierarchicalTable } from "components/table/HierarchicalTable";
```

Comment:
Please use `Table` instead of this component, it's marked for removal https://github.com/SUSE/spacewalk/issues/28408. Now that I look at it, this should be marked as `@deprecated`, I will go do that now.